### ISSUE TITLE
Extended, Custom Post Type reverts to using the Pods, label instead of registered Custom Post Type values for labels and rewrite slug

### ISSUE THREAD

**Issue by agileware-justin** [pods-framework/pods#6348]
### Description

Using a custom to the website plugin that registers a custom post type, "aa-event" which has a Label of "Events" and "Event" and Slug of "events". This has been extended using Pods to add some additional fields.

After updating to 2.8.7, this Custom Post Type reverts to using the Pods, label "aa-event" and removes the Slug.

Reverting to 2.8.6 does not solve the issue.
Reverting to 2.8.5 solves the issue.

Agileware Ref: PROJ-2135

### Version

2.8.7

### Testing Instructions

1. Register a custom post type, Events in a custom plugin as shown below.
2. Use Pods to extend the custom post type and additional fields, see Pods migrate export below.

```
function agileware_events_create_post_type() {
  register_post_type('aa-event', array(
     'labels' => array(
      'name' => __('Event', 'textdomain' ),
      'singular_name' => __('Event', 'textdomain' ),
    ),
    'public' => true,
    'has_archive' => false,
    'show_ui' => true,
    'rewrite' => array('slug' => 'events'),
    'supports' => array(
      'title',
      'editor',
      'excerpt',
      'thumbnail',
      'author',
      'revisions'
    ),
  ));
}
```

### Pods 2.8.5

![Screenshot_20211203_103843](https://user-images.githubusercontent.com/58866555/144520337-9e6c8a6e-097a-45ac-b95b-29723d5c29fe.png)

### Pods 2.8.7

![Screenshot_20211203_104015](https://user-images.githubusercontent.com/58866555/144520166-5a6b8055-43e3-4479-8527-b938b4787b8d.png)
)

### Possible Workaround

Revert to Pods 2.8.5

### Pods Package

```json
{
    "meta": {
        "version": "2.8.5",
        "build": 1638487816
    },
    "pods": [
        {
            "name": "aa-event",
            "id": 4899,
            "label": "Aa-event",
            "description": "",
            "type": "post_type",
            "storage": "meta",
            "object": "aa-event",
            "pfat_enable": "0",
            "pfat_run_outside_loop": "0",
            "pfat_single": "Single Grant Template",
            "pfat_append_single": "append",
            "pfat_filter_single": "the_content",
            "pfat_append_archive": "append",
            "pfat_filter_archive": "custom",
            "rest_enable": "0",
            "rest_base": "aa-event",
            "read_all": "0",
            "write_all": "0",
            "pfat_filter_archive_custom": "the_content",
            "_migrated_28": "1",
            "built_in_taxonomies_region": "1",
            "built_in_taxonomies_cohort": "0",
            "built_in_taxonomies_domain": "0",
            "groups": [
                {
                    "name": "more_fields",
                    "id": 21344,
                    "label": "More Fields",
                    "description": "",
                    "weight": 0,
                    "fields": [
                        {
                            "name": "organiser",
                            "id": 4929,
                            "label": "Organiser",
                            "description": "",
                            "weight": 0,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "aa_event_start",
                            "id": 12876,
                            "label": "Start Date",
                            "description": "",
                            "weight": 1,
                            "type": "date",
                            "required": "0",
                            "date_type": "wp",
                            "date_format": "fjy",
                            "date_allow_empty": "1",
                            "date_html5": "1",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "date_repeatable": "0"
                        },
                        {
                            "name": "aa_event_start_time",
                            "id": 4946,
                            "label": "Start Time",
                            "description": "",
                            "weight": 2,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "aa_event_end",
                            "id": 12877,
                            "label": "End Date",
                            "description": "",
                            "weight": 3,
                            "type": "date",
                            "required": "0",
                            "date_type": "wp",
                            "date_format": "fjsy",
                            "date_allow_empty": "1",
                            "date_html5": "1",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "date_repeatable": "0"
                        },
                        {
                            "name": "aa_event_end_time",
                            "id": 4947,
                            "label": "Finish Time",
                            "description": "",
                            "weight": 4,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "pricing",
                            "id": 4939,
                            "label": "Pricing",
                            "description": "",
                            "weight": 5,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "address",
                            "id": 4930,
                            "label": "Address",
                            "description": "",
                            "weight": 6,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "suburb",
                            "id": 4931,
                            "label": "Suburb",
                            "description": "",
                            "weight": 7,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "state",
                            "id": 4932,
                            "label": "State",
                            "description": "",
                            "weight": 8,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "postcode",
                            "id": 4933,
                            "label": "Postcode",
                            "description": "",
                            "weight": 9,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "contact_first_name",
                            "id": 4934,
                            "label": "Contact First Name",
                            "description": "",
                            "weight": 10,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "contact_last_name",
                            "id": 5738,
                            "label": "Contact Last Name",
                            "description": "",
                            "weight": 11,
                            "type": "text",
                            "required": "0",
                            "unique": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "phone_number",
                            "id": 4935,
                            "label": "Phone Number",
                            "description": "",
                            "weight": 12,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "fax_number",
                            "id": 4936,
                            "label": "Fax Number",
                            "description": "",
                            "weight": 13,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "email",
                            "id": 4937,
                            "label": "Email",
                            "description": "",
                            "weight": 14,
                            "type": "email",
                            "required": "0",
                            "email_max_length": "255",
                            "email_html5": "0",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "email_repeatable": "0"
                        },
                        {
                            "name": "website",
                            "id": 4938,
                            "label": "Website",
                            "description": "",
                            "weight": 15,
                            "type": "website",
                            "required": "0",
                            "website_format": "normal",
                            "website_clickable": "1",
                            "website_new_window": "1",
                            "website_max_length": "255",
                            "website_html5": "0",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "website_repeatable": "0",
                            "website_allow_port": "0"
                        },
                        {
                            "name": "member_number",
                            "id": 5016,
                            "label": "Member Number",
                            "description": "",
                            "weight": 16,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "role",
                            "id": 5327,
                            "label": "User role",
                            "description": "",
                            "weight": 17,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "payment_status",
                            "id": 5317,
                            "label": "Payment Status",
                            "description": "",
                            "weight": 18,
                            "type": "text",
                            "required": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "1",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "text_repeatable": "0"
                        },
                        {
                            "name": "transaction_id",
                            "id": 5721,
                            "label": "eWay Transaction ID",
                            "description": "",
                            "weight": 19,
                            "type": "text",
                            "required": "0",
                            "unique": "0",
                            "text_allow_shortcode": "0",
                            "text_allow_html": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "1",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "text_repeatable": "0"
                        }
                    ]
                }
            ]
        }
    ]
}
```
```

**Comment by JoryHogeveen** [comment]
Duplicate of #6342 and fixed in 2.8.8: https://github.com/pods-framework/pods/archive/refs/heads/release/2.8.8.zip

**Comment by agileware-justin** [comment]
@JoryHogeveen Thanks for confirming and linking up with the fix. Happy holidays! 😄
